README.md

```diff
@@ -57,6 +58,22 @@ client = Bitreserve::Client.new
 puts client.all_tickers
 ```
 
+# Options
+
+This is a summary of the supported options when instantiating a new client, and their default values:
+
+```ruby
+Bitreserve::Client.new(
+  # bearer_token for OAuth authentication
+  token: ENV['BITRESERVE_AUTH_TOKEN']
+
+  # use the sandbox api (https://api-sandbox.bitreserve.org) instead of the real one
+  sandbox: false
```

Comment:
Hum... The main issue with this is that if you want to create more than one client you might have to send the options more than once. However, it does allow more flexibility for the same exact reason. I guess those are the biggest tradeoffs, just want to make sure they are understood and that we make a choice with that in mind.

So, my question is, what do we want to favor, ease to create multiple clients, or ease to make each client different from the other?